pkg/grpc/server.go

```diff
@@ -129,8 +129,10 @@ func (s *server) getMiddlewareOptions() []grpc_go.ServerOption {
 	opts = append(
 		opts,
 		grpc_go.StreamInterceptor(diag.TracingGRPCMiddlewareStream(s.tracingSpec)),
-		grpc_go.UnaryInterceptor(diag.TracingGRPCMiddlewareUnary(s.tracingSpec)),
-		grpc_go.StatsHandler(diag.DefaultGRPCMonitoring.ServerStatsHandler))
+		grpc_go.UnaryInterceptor(diag.TracingGRPCMiddlewareUnary(s.tracingSpec)))
```

Comment:
Should be added only if tracing is enabled

Reply:
We will handle this in the different issue - #1373